### PR TITLE
[CI:DOCS] clarify ps(1) fallback of `podman top`

### PR DESCRIPTION
**Pull request by vrothberg**
Podman top falls back to executing ps(1) inside the container in the
presence of ps-specific flags.  Clarify that a bit more to help users
resolve issues when, for instance, ps(1) isn't installed in the
container.

Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
Note to self: close https://github.com/containers/podman/pull/8449 after merge

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat PTAL
@containers/podman-maintainers PTAL

**Review by saschagrunert — APPROVED** [review]
LGTM

**Pull request merged**
